.gitignore

```diff
@@ -12,7 +12,6 @@ yarn-error.log
 
 # Xcode
 #
-build/
```

Comment:
why this change? the build artifacts should not be in git (the config plugin build included)